### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Initial implementation

## why
* Provision Service Control Policies (SCP) for AWS Organizations and Organizational Units

### CODE REVIEW

**Review by Gowiem — DISMISSED** [review]
Looks great @aknysh!

File: main.tf [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
   }
 }
 
-locals {
-  example = format("%v %v", var.example, join("", random_integer.example[*].result))
+resource "aws_organizations_policy" "this" {
+  count       = module.this.enabled ? 1 : 0
+  name        = module.this.id
+  description = var.service_control_policy_description
+  content     = jsonencode(local.service_control_policy_content)
```
Comment (osterman) [inline main.tf:12]: We should use an `aws_iam_policy_document` here so that we get validation.

Comment (osterman) [reply main.tf:12]: Unfortunatly this is not possible since we cannot `for-each` on the dynamic statements in a `data resource`

Comment (aknysh) [reply main.tf]: updated to use `aws_iam_policy_document`

**Review by osterman — APPROVED** [review]

**Pull request merged**